src/components/calcite-shell-panel/calcite-shell-panel.scss

```diff
@@ -1,26 +1,49 @@
-$shell-panel-width: 20vw !default;
-$shell-panel-min-width: 240px !default;
-$shell-panel-max-width: 360px !default;
-
 :host {
-  height: 100%;
   position: relative;
   display: flex;
-  background-color: var(--calcite-app-background-content);
+  align-items: stretch;
+  background-color: var(--calcite-app-background-transparent);
+  pointer-events: none;
+
+  --calcite-app-shell-panel-width: 20vw;
+  --calcite-app-shell-panel-min-width: 240px;
+  --calcite-app-shell-panel-max-width: 360px;
+  --calcite-app-shell-panel-max-height-small: 35vh;
+  --calcite-app-shell-panel-max-height-medium: 55vh;
+  --calcite-app-shell-panel-max-height-large: 85vh;
 }
 
 .content {
-  width: $shell-panel-width;
-  min-width: $shell-panel-min-width;
-  max-width: $shell-panel-max-width;
+  align-items: stretch;
+  align-self: stretch;
+  background-color: var(--calcite-app-background-content);
+  display: flex;
+  width: var(--calcite-app-shell-panel-width);
+  min-width: var(--calcite-app-shell-panel-min-width);
+  max-width: var(--calcite-app-shell-panel-max-width);
   border-left: 1px solid var(--calcite-app-border);
   border-right: 1px solid var(--calcite-app-border);
   padding: 0;
-  margin: 0;
-  height: 100%;
+  pointer-events: auto;
+}
+
+.content--detached {
+  border: 1px solid var(--calcite-app-border);
+  border-radius: var(--calcite-app-border-radius);
+  box-shadow: var(--calcite-app-shadow-0);
+  margin: var(--calcite-app-cap-spacing-half) var(--calcite-app-side-spacing-half) auto;
+  max-height: var(--calcite-app-shell-panel-max-height-medium);
   overflow: auto;
 }
 
+:host([detached-scale="s"]) .content--detached {
+  max-height: var(--calcite-app-shell-panel-max-height-small);
+}
+
+:host([detached-scale="l"]) .content--detached {
```

Comment:
nitpick: This could be adjusted so there is a modifier on the content element to avoid the descendent selector.